src/test/java/com/endava/service/IncomeExpenseServiceImplTest.java

```diff
@@ -193,15 +189,15 @@ public void testUpdateIncomeIncorrectDate() {
 
     @Test
     public void testViewIncomesOnCorrectDate() {
-        List<Income> incomes = new ArrayList<>( Arrays.asList(income1, income2, income3, income4) );
+        List<MoneyTransfer> moneyTransfers = new ArrayList<>( Arrays.asList(moneyTransfer, moneyTransfer1, moneyTransfer2, moneyTransfer3) );
```

Comment:
You don't need new ArrayList here. Arrays.asList returns unmodifiable List, right what you need :)